txcontext/tx_context.go

```diff
@@ -64,4 +64,8 @@ type OutputState interface {
 	// GetStateHash returns expected State Hash. This is only used
 	// by Eth JSON tests and can be ignored for most implementations.
 	GetStateHash() common.Hash
+
+	// GetLogsHash returns expected rlp-encoded Logs Hash. This is only used
```

Comment:
usually an interface is designed for an open-ended number of clients, i.e. documentation should not say where it is used, and that it is not necessary in other cases.